### PR TITLE
Yelp API where to eat foods

### PR DESCRIPTION
**Pull request by Suabd**
Hello!
This is YELP API for foods to eat. Let me know if it needs any change, Thanks!

### CODE REVIEW

**Comment by claraj** [comment]
Thank you. Consider how this will be linked in to the rest of the project. Other parts of the program need to call the code in this module, and they'll need to access the data that this module is fetching from Yelp.  At some point, you'll print that data, but you also may need to save it to the database, and how would you do that with this code? 

Another part of the code is going to handle getting user input for the city name.

So instead of a main function that gets user input, this module will have a function that receives a location, calls function(s) to make APIs call, then returns that data to the rest of the program. 

```

def get_restaurants_for_location(location):
    
    # call function(s) that make API call
    # return data about restaurants 

```

You'll be able to use a lot of the code you already have. 

Somewhere else in the program will get the location, then call this new function, and then present that data to the user; also use that data if the user wishes to save all the data from all 3 APIs.

File: main.py [hunk 1]
```diff
@@ -1,5 +1,27 @@
 """
-Project 3 - A Useful Travel App (working title)
+This is main section
+"""
+from ui import message
+from api_food_to_eat import get_restaurants_for_location
+
+def main():
+    city = input('Enter the name of the city: ').strip()
+    country = input('Enter the 2-letter country code: ').strip().upper()
```
Comment (sndmnska) [inline main.py:9]: `input()` functions should be replaced with a call to ui.py. 

```
from ui import user_question

city = user_question('Enter the name of the city:  ').strip()
```

**Review by sndmnska — COMMENTED** [review]
Looks great so far! Thanks for putting some work into `main.py` as well.   I do have a couple notes.

- Could you make it so that you return one restaurant from your API call?    How you do that is up to you.  It should return as a string of one restaurant name.  
- I also made some changes to ui.py in another branch, which can help us keep `input()` in the ui module. Once that's been merged, you can update main.py with the appropriate methods. 

Otherwise, it looks great!

**Review by sndmnska — APPROVED** [review]
Gave a quick correction from input() to ui.user_question(), otherwise looking good!

**Review by JoshuaMathews — APPROVED** [review]
Seems good? Unsure about conflicts but will fix if something goes wrong.

**Pull request merged**